Kino/MainScreen/View/DetailsView/DetailsViewController.swift

```diff
@@ -90,6 +300,18 @@ class DetailsViewController: UIViewController {
         contentView.addSubview(trailerImageView)
         contentView.addSubview(playButton)
         contentView.addSubview(playLabel)
+        contentView.addSubview(roundedView)
+        
+        roundedView.addSubview(movieTitleLabel)
+        roundedView.addSubview(starImageView)
+        roundedView.addSubview(ratingLabel)
+        roundedView.addSubview(bookmarkButton)
+        roundedView.addSubview(genresCollectionView)
+        roundedView.addSubview(infoStackView)
+        roundedView.addSubview(descriptionLabel)
+        roundedView.addSubview(movieDescriptionLabel)
+        roundedView.addSubview(headerView)
```

Comment:
Add extension with addSubviews method and replace here. I think you can combine both fillUIFromViewModel() and configureUI methods into one.

Reply:
done